src/test/java/edu/harvard/iq/dataverse/pidproviders/PidUtilTest.java

```diff
@@ -133,6 +145,7 @@ public static void setUpClass() throws Exception {
         pidProviderFactoryMap.put(HandlePidProvider.TYPE, new HandleProviderFactory());
         pidProviderFactoryMap.put(FakeDOIProvider.TYPE, new FakeProviderFactory());
         pidProviderFactoryMap.put(EZIdDOIProvider.TYPE, new EZIdProviderFactory());
+        pidProviderFactoryMap.put(CrossRefDOIPidProvider.TYPE, new CrossRefDOIProviderFactory());
```

Comment:
My guess is that the test changes don't yet do anything because the code above doesn't look for jvm options with crossref1 in them. Once those changes are made, I think adding to the tests here to assure that a crossref doi - one with authority 10.5072 and shoulder FK2/ as currently defined, is recognized by a provider with the id crossref1.

Reply:
add that to JvmSettingsTest.java